processor/src/main/java/org/mapstruct/ap/internal/model/source/FormattingParameters.java

```diff
@@ -40,4 +40,28 @@ public String getNumber() {
         return number;
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
True to say I blindly copypasted this from the changeset you linked earlier. I will remove it and check that all tests still pass.